vision/cloud-client/detect/beta_snippets.py

```diff
@@ -174,6 +174,99 @@ def detect_handwritten_ocr_uri(uri):
 # [END vision_handwritten_ocr_gcs_beta]
 
 
+# [START vision_fulltext_detection_pdf_beta]
+def detect_document_features(path):
+    """Detects document features in a PDF/TIFF/GIF file.
```

Comment:
i was under the impression that **document features** includes not only paragraphs/words/symbols but also stuff like tables or images.

Reply:
Possibly. I did not have a good PDF to test that out.